gnosis/eth/ethereum_client.py

```diff
@@ -1547,6 +1504,43 @@ def estimate_data_gas(data: bytes):
                 gas += GAS_CALL_DATA_BYTE
         return gas
 
+    def estimate_fee_eip1159(
+        self, tx_speed: TxSpeed = TxSpeed.NORMAL
+    ) -> Tuple[int, int]:
+        """
+        :return: Tuple[BaseFeePerGas, MaxPriorityFeePerGas]
+        :raises: ValueError if not supported on the network
+        """
+        if tx_speed == TxSpeed.SLOWEST:
+            percentile = 0
+        elif tx_speed == TxSpeed.VERY_SLOW:
+            percentile = 10
+        elif tx_speed == TxSpeed.SLOW:
+            percentile = 25
+        elif tx_speed == TxSpeed.NORMAL:
+            percentile = 50
+        elif tx_speed == TxSpeed.FAST:
+            percentile = 75
+        elif tx_speed == TxSpeed.VERY_FAST:
+            percentile = 90
+        elif tx_speed == TxSpeed.FASTEST:
+            percentile = 100
+
+        result = self.w3.eth.fee_history(1, "latest", reward_percentiles=[percentile])
```

Comment:
Maybe add a comment to https://github.com/ethereum/execution-apis/blob/main/src/eth/fee_market.json#L15 as this is a quite new rpc endpoint, and helps understand this logic